regression_tests/pull_request_tests.sh

```diff
@@ -3,16 +3,22 @@
 # exit early if something goes wrong
 set -e
 
+export TARGET_ENVIRONMENT=${TARGET_ENVIRONMENT:-toolspr}
```

Comment:
The use of this via the AWS Console to fire off regression tests using a different target environment needs adding to the README.

Reply:
Btw I ran into an issue with that because my env isn't part of a pipeline